hack/release.sh

```diff
@@ -39,7 +39,7 @@ export DOCKER_REPO_OVERRIDE=DOCKER_NOT_SET
 readonly YAML_LIST="$(mktemp)"
 $(dirname $0)/generate-yamls.sh "${REPO_ROOT_DIR}" "${YAML_LIST}"
 readonly YAMLS_TO_PUBLISH=$(cat "${YAML_LIST}" | tr '\n' ' ')
-readonly RELEASE_YAML="$(head -n1 ${YAML_LIST})"
```

Comment:
Please revert the changes in this file, or it will break the v0.2 branch.

Reply:
Done, forgot about this one.